Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/ActionsFactory.cs

```diff
@@ -95,7 +97,7 @@ static Act GenerateSelectedAction(Act selectedAction, Context mContext)
             else
             {
                 Act instance = (Act)selectedAction.CreateCopy();
-                if (selectedAction is IObsoleteAction)
+                if (selectedAction is IObsoleteAction && IsLegacyTabSelected)
                 {
```

Comment:
please explain why IsLegacyTabSelected was added and IObsoleteAction was not enough

Reply:
There exist some actions which implements IObsoleteAction and aren't  listed under Legacy Actions. like WindowsAction on Java Platform implements IObsoleteAction but still is listed under Platform Actions and that's the reason Ravi reported an issue where,"LegacyAction Warning Pop Up is being generated on adding Windows Action over Java Platform"